### PR TITLE
Add new families

### PR DESCRIPTION
**Pull request by ddomingof**
Add new families:

-  5-HT
- AGTR
- CYP

### CODE REVIEW

File: relations.csv [hunk 1]
```diff
@@ -4067,3 +4067,80 @@ HGNC,MIRLET7A3,isa,FPLX,MIRLET7A
 FPLX,MIRLET7F,isa,FPLX,MIRLET7
 HGNC,MIRLET7F1,isa,FPLX,MIRLET7F
 HGNC,MIRLET7F2,isa,FPLX,MIRLET7F
+HGNC,HTR1A,isa,FPLX,5-HT
+HGNC,HTR1B,isa,FPLX,5-HT
+HGNC,HTR1D,isa,FPLX,5-HT
+HGNC,HTR1E,isa,FPLX,5-HT
+HGNC,HTR1F,isa,FPLX,5-HT
+HGNC,HTR2A,isa,FPLX,5-HT
```
Comment (cthoyt) [inline relations.csv:4075]: could you make more fine granular families like HTR1, HTR2, HTR3, HTR4, and HTR5 that go in the middle?

**Comment by steppi** [comment]
Thanks @ddomingof, I'll take it from here.

**Comment by ddomingof** [comment]
@steppi perfect! thanks

**Comment by steppi** [comment]
Hi @ddomingof . I'm unable to push commits to your PR branch. Is "Allow edits from maintainers" checked on your PR?

**Comment by ddomingof** [comment]
@steppi yes it is, weird :/

**Comment by steppi** [comment]
That is weird. It must be something on our side

**Comment by bgyori** [comment]
The issue is that @ddomingof you forked johnbachman/famplex instead of sorgerlab/famplex, which is understandable due to how historically this repo was created. But we may need to close this and open a new PR, and you might want to re-fork the repo from sorgerlab. Thanks!

**Comment by ddomingof** [comment]
@bgyori sounds good! Thanks!

**Pull request closed without merging**